spec/unit/sup_spec.rb

```diff
@@ -16,6 +16,21 @@
           )
       end
 
+      it 'run hab sup with a single peer' do
+        expect(chef_run).to run_hab_sup('single_peer').with(
+          override_name: 'single_peer',
+          peer: ['127.0.0.2']
```

Comment:
To ensure that the logic of using a string vs an array is sound, how  about `peer: '127.0.0.2'`?

Reply:
When you pass in a single string instead of an array the resource converts it into a single item array on load. It works the same way with the bind parameter. The chefspec test is validating the post-load converted array. If we change get that behavior we should probably change it similarly in bind

Reply:
it looks like binding is only tested in the functional inspec tests and not in unit tests
https://github.com/chef-cookbooks/habitat/blob/master/test/integration/service/default_spec.rb#L26

to be clear I'm fine with re-doing the expectations on the interface in the tests. I suggest that we add similar tests and expectations to bind either way. What's your take @jtimberman?